src/main/groovy/com/bmuschko/gradle/docker/tasks/AbstractDockerRemoteApiTask.groovy

```diff
@@ -167,6 +172,17 @@ abstract class AbstractDockerRemoteApiTask extends DefaultTask {
         dockerClient
     }
 
+    private DockerCmdExecFactory getDockerCmdExecFactory() {
+        boolean useNettyExecFactory = Boolean.valueOf(getProject().findProperty('gdpNettyExecFactory').toString())
```

Comment:
I am surprised that the method call of `toString()` doesn't fail if the `findProperty` method return null. You will likely want to catch that with via `?.toString()`.

Reply:
That's interesting that did not fail. Maybe some groovy magic? IDK. I'll put in this fix.